.travis.yml

```diff
@@ -8,6 +8,7 @@ php:
 
 env:
   - SYMFONY_VERSION="~2.3||~3.0"
+  - SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
this is wrong (as it'll now just add one new build with only this env set). You want something like this:
```
env:
  matrix:
    - SYMFONY_VERSION="~2.3|~3.0"
  global:
    - SYMFONY_DEPRECATIONS_HELPER=weak
```